composer.json

```diff
@@ -21,6 +26,13 @@
         "symfony/doctrine-bridge": "^3.4 || ^4.0",
         "symfony/framework-bundle": "^3.4 || ^4.0"
     },
+    "conflict": {
+        "doctrine/annotations": "< 1.7.0",
+        "jackalope/jackalope": "< 1.3.1",
+        "phpcr/phpcr-shell": "< 1.0.0-beta1",
+        "symfony/dependency-injection": "4.0.0",
+        "twig/twig": "< 1.42.3"
```

Comment:
1.42.3 contains a fix for php 7.4

Reply:
maybe do `composer require twig/twig:>1.42.3` in the `CI` instead of conflicting all older versions?

Reply:
hm, that could make sense.

on the other hand, twig released version 3 recently, and most people updating their stuff will be on twig 2 at least. i don't expect there to be many changes in twig 1.x versions anymore, so upgrading to 1.42.3 should not be a big issue. or do you think it is a problem?

Reply:
Yeah twig updates are mostly straight forward and mostly not a big deal.